1_slices_modules_and_objects/exercises/acronym/src/lib.rs

```diff
@@ -10,7 +10,7 @@
 /// ```
 // Side note: this doc example is also tested when running `cargo test`
 pub fn make_acronym(name: &str) -> String {
-    unimplemented!()
+    name.split_whitespace().map(|word| word.chars().nth(0).unwrap().to_ascii_uppercase()).collect::<String>()
```

Comment:
Great, the advantage with this is that you avoid maybe allocating a new String. For small names, a `String::from(name)` will have very little cost (basically strcpy of a few bytes) but as soon as your name becomes too big, the String is created on the heap and thus requires dynamic memory allocation